src/config/layers.ts

```diff
@@ -215,6 +215,7 @@ const qFaultsConfig: LayerProps = {
     options: {
         sublayers: [
             {
+                title: 'Quaternary_Faults',
```

Comment:
this needs to match the name in the layer that we get back from the [fetch request](https://github.com/UGS-GIO/geohaz-v2/pull/16/files#diff-003ef9ba2137b028fa5739ee164e167f1a3f8bcbf8c735a36d3fddb2136bfa63R86-R94)